rio_tiler/colormap.py

```diff
@@ -133,6 +139,67 @@ def apply_discrete_cmap(
     return data[:-1], data[-1]
 
 
+def parse_color(value: Union[List[int], str]) -> List[int]:
```

Comment:
Should the type be a tuple because input is always length 3 or 4 and output is always length 4?